templates/tfengine/recipes/org/folder.hcl

```diff
@@ -12,6 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+data = {
+	parent = "organizations/{{.org_id}}"
```

Comment:
Doesn't look like this parent field is needed by terragrunt. Was this just for your testing?

Reply:
That's correct. I looked for a place where I can pull it out and have it be used by both recipes, but I couldn't find one, so I just pulled it out in an arbitrary recipe.

Reply:
It is generally better to put data in a template if possible as it is clearer. The main appeal of top level data block is to avoid having to duplicate across many resources. If there is no example of this today I would prefer not manufacturing one just to test this behaviour. As we develop more templates or clean up our recipes we can use this where appropriate.

Reply:
I'll remove it for now then. But it did work when I tested it.